package.json

```diff
@@ -18,9 +18,10 @@
   ],
   "scripts": {
     "eslint": "eslint bin tests",
-    "test": "npm run eslint && npm run test:coverage",
+    "test": "npm run eslint && npm run coverage",
     "test:unit": "jasmine --config=tests/spec/unit.json",
-    "test:coverage": "nyc jasmine --config=tests/spec/coverage.json"
+    "coverage": "nyc npm run test:unit",
+    "coverage:report": "nyc report -r text-lcov > coverage.lcov && codecov"
```

Comment:
We already have an lcov reporter configured below, so we should not need to convert anything, right?

Reply:
yes we do not need. removed.